odoo/addons/base/models/ir_translation.py

```diff
@@ -261,7 +262,8 @@ def _auto_init(self):
         res = super(IrTranslation, self)._auto_init()
         # Add separate md5 index on src (no size limit on values, and good performance).
         tools.create_index(self._cr, 'ir_translation_src_md5', self._table, ['md5(src)'])
-        tools.create_index(self._cr, 'ir_translation_ltn', self._table, ['name', 'lang', 'type'])
+        tools.create_unique_index(self._cr, 'ir_translation_unique', self._table,
+                                  ['type', 'name', 'lang', 'res_id', 'md5(src)'])
```

Comment:
why `md5(src)`?

Reply:
it is rebased from odoo-dev/odoo@5c40de7b79cbab6203e69e8b1f1697d8d7252eee